### ISSUE TITLE
units for hsl(a)

### ISSUE THREAD

**Issue by Reasonable-Solutions** [SentiaAnalytics/bs-css#139]
currently the type of the hsl(a) functions is
` Int -> Int -> Int -> Float` and the implementation is 
```
  let string_of_hsla = (h, s, l, a) =>
    "hsla("
    ++ string_of_int(h)
    ++ ", "
    ++ string_of_int(s)
    ++ "%, "
    ++ string_of_int(l)
    ++ "%, "
    ++ string_of_float(a)
    ++ ")";
```
The problem with this approach is that some colors are at, e.g a saturation of `95.5%` and with string_of_int that gets kinda hard to express. Could the `s` and `l` perhaps be [> `percent(float)] ?

**Comment by giraud** [comment]
yes, definition is incorrect.
It'll be  angle -> float -> float -> num(float) | percent(float) :
```
hsl(deg(7.), 8., 9.)
hsla(deg(10.), 11., 12., `num(0.5))
hsla(rad(4.7), 11., 12., pct(50.))
```

**Comment by giraud** [comment]
fixed in 9.0.1
